hack/logcheck.conf

```diff
@@ -20,6 +20,8 @@ structured k8s.io/kubernetes/cmd/kubelet/.*
 structured k8s.io/kubernetes/pkg/kubelet/.*
 structured k8s.io/kubernetes/pkg/proxy/.*
 structured k8s.io/kubernetes/pkg/scheduler/.*
+structured k8s.io/kms/.*
+structured k8s.io/apiserver/pkg/storage/value/encrypt/.*
```

Comment:
```suggestion
structured k8s.io/apiserver/pkg/storage/value/.*
```

Reply:
I had omitted those because I thought they need to be fixed but seems `k8s.io/apiserver/pkg/server/options/encryptionconfig` and `k8s.io/apiserver/pkg/storage/value` have structured logging mostly except the ones we're fixing. I've enabled it for those now.